try:
    import test0
except Exception as e:
    print(e)
    print(type(e))
    err = e